src/pkg/gc/gc_test.go

```diff
@@ -161,11 +163,14 @@ func generateFiveTestContainers() (string, map[string]string) {
 	twoWeekOld := timeNow.Add(2 * -7 * 24 * time.Hour)
 
 	idsAndDatesMap := make(map[string]string)
-	idsAndDatesMap["8dfafdbc3a40"] = timeNow.Format(time.RFC3339)
-	idsAndDatesMap["9cd87474be90"] = fiveMinutesOld.Format(time.RFC3339)
-	idsAndDatesMap["3176a2479c92"] = twelweHoursOld.Format(time.RFC3339)
-	idsAndDatesMap["4cb07b47f9fb"] = weekOld.Format(time.RFC3339)
-	idsAndDatesMap["5c76a2479c92"] = twoWeekOld.Format(time.RFC3339)
+	for i := 1; i <= amount; i++ {
+		extra, _ := time.ParseDuration(fmt.Sprintf("%ds", i))
```

Comment:
I don't think we should eat this error. Instead we could pass `*testing.T` into this function and do `t.Fatalf()` if (for some extremely unlikely reason) ParseDuration() fails - as it could be very painful to debug.